pyscf/adc/radc.py

```diff
@@ -1026,19 +1030,21 @@ def get_imds_ea(adc, eris=None):
             M_ab += 0.25*lib.einsum('lmaf,lmfb->ab',t2_1, temp_t2, optimize=True)
             M_ab -= lib.einsum('mlaf,mlbf->ab',t2_1, temp_t2, optimize=True)
 
-            temp_vvvv_t2 = np.dot(eris_vvvv,t2_1_r.T)
-            temp_vvvv_t2 = temp_vvvv_t2.reshape(nvir,nvir,nocc,nocc)
-            M_ab += 0.25*lib.einsum('adlm,mlbd->ab',temp_vvvv_t2, t2_1, optimize=True)
-            M_ab -= 0.25*lib.einsum('adlm,lmbd->ab',temp_vvvv_t2, t2_1, optimize=True)
-            M_ab -= 0.25*lib.einsum('adml,mlbd->ab',temp_vvvv_t2, t2_1, optimize=True)
-            M_ab += 0.25*lib.einsum('adml,lmbd->ab',temp_vvvv_t2, t2_1, optimize=True)
-
             M_ab -= 0.25*lib.einsum('mlad,mlbd->ab', temp_t2, t2_1, optimize=True)
             M_ab += 0.25*lib.einsum('mlad,lmbd->ab', temp_t2, t2_1, optimize=True)
             M_ab += 0.25*lib.einsum('lmad,mlbd->ab', temp_t2, t2_1, optimize=True)
             M_ab -= 0.25*lib.einsum('lmad,lmbd->ab', temp_t2, t2_1, optimize=True)
             M_ab -= lib.einsum('mlad,mlbd->ab', temp_t2, t2_1, optimize=True)
+            del temp_t2
 
+            temp_vvvv_t2 = adc.t2_vvvv_imds.T
```

Comment:
Defining temp_vvvv_t2 can be avoided if you simply swap indices in the einsums below. Please fix.